File: xsolla-store-sdk/src/main/java/com/xsolla/android/store/entity/response/items/PhysicalItemsResponse.java
package com.xsolla.android.store.entity.response.items;

import com.google.gson.annotations.SerializedName;
import com.xsolla.android.store.entity.response.common.Group;
import com.xsolla.android.store.entity.response.common.Price;
import com.xsolla.android.store.entity.response.common.VirtualPrice;

import java.util.List;

public class PhysicalItemsResponse {
    private List<Item> items;

    public List<Item> getItems() {
        return items;
    }

    public class Item {
        private String sku;
        private String name;
        private List<Group> groups;
        private List<Object> attribites;
        private String type;
        private String description;

        @SerializedName("image_url")
        private String imageUrl;

        @SerializedName("is_free")
        private boolean isFree;
        private Price price;

        @SerializedName("virtual_prices")
        private List<VirtualPrice> virtualPrices;

        public String getSku() {
            return sku;
        }

        public String getName() {
            return name;
        }

        public List<Group> getGroups() {
            return groups;
        }

        public List<Object> getAttribites() {
            return attribites;
        }

        public String getType() {
            return type;
        }

        public String getDescription() {
            return description;
        }

        public String getImageUrl() {
            return imageUrl;
        }

        public boolean isFree() {
            return isFree;
        }

        public Price getPrice() {
            return price;
        }

        public List<VirtualPrice> getVirtualPrices() {
            return virtualPrices;
        }

    }
}

File: xsolla-store-sdk/src/main/java/com/xsolla/android/store/entity/response/items/PhysicalItemsResponse.kt
package com.xsolla.android.store.entity.response.items

import com.google.gson.annotations.SerializedName
import com.xsolla.android.store.entity.response.common.Group
import com.xsolla.android.store.entity.response.common.Price
import com.xsolla.android.store.entity.response.common.VirtualPrice

data class PhysicalItemsResponse(val items: List<Item> = emptyList()) {
    data class Item(
        val sku: String,
        val name: String,
        val groups: List<Group> = emptyList(),
        val attributes: List<Any> = emptyList(),
        val type: String,
        val description: String,
        @SerializedName("image_url")
        val imageUrl: String,
        @SerializedName("is_free")
        val isFree: Boolean,
        val price: Price,
        @SerializedName("virtual_prices")
        val virtualPrices: List<VirtualPrice> = emptyList()
    )
}